test/test_party_controller.py

```diff
@@ -350,6 +350,22 @@ def test_get_party_by_survey_id_and_enrolment_statuses_with_invalid_enrolment(se
             "B", mock_business["id"], DEFAULT_SURVEY_UUID, ["ENABLED", "PENDING"]
         )
 
+    def test_get_latest_business_details(self):
```

Comment:
I've followed the current test set-up, I don't like it, but in this instance I decided not to change it